lib/client.js

```diff
@@ -2502,6 +2568,10 @@ function checkTurnServers(client) {
     if (!client._supportsVoip) {
         return;
     }
+    if (client.isGuest()) {
+        return; // guests can't access TURN servers
```

Comment:
This comment probably isn't really about the JS SDK but this surprises me. I have queried this OOB.